### PR TITLE
trezor-agent: 0.9.0 -> 0.9.2

### PR DESCRIPTION
**Pull request by jb55**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27603,23 +27603,25 @@ EOF
   trezor_agent = buildPythonPackage rec{
     name = "${pname}-${version}";
     pname = "trezor_agent";
-    version = "0.9.0";
+    version = "v0.9.2";
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: if you need the `v` you can add it to `rev`. It shouldn't be here because this is part of the name.

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by jb55** [comment]
ah thanks I always forget that. I've pushed a fix

**Comment by jb55** [comment]
a bunch of things changed since the original PR which simplified it a bunch! I rebased on master.

**Pull request closed without merging**